modules/preview-react/text/stories/Text.stories.mdx

```diff
@@ -0,0 +1,131 @@
+
+<Meta title="Preview/Text/React" />
+
+# Canvas Kit React Text
+
+`Text` is a group of text components that provide a common, ergonomic API to render text with
+simplified usage of Canvas Kit type token.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-react
+```
+
+## Text
+
+### Usage
+
+`Text` is a basic text component to render modified text. `Text` component composes `Box` component
+that allows to use all layout props, such as `display`, `margin`, `lineHeight` etc. It also has
+their own props such as `level` and `size` that should be provided together, because they refer to
+type token level and size, and `variant` to set variant color.
+
+By default `Text` component is a `span` element, but it's possible to override the element by
+passing `as` prop.
+
+```tsx
+<Text as="p" fontSize={14} fontWeight="medium">
+  Text here...
+</Text>
+```
+
+<ExampleCodeBlock code={BasicText} />
+
+### Text Props
+
+As previously mentioned, `Text` is built on top of `Box` and has access to all its props. In
+addition `Text` has its own props:
+
+<ArgsTable of={TextComponent} />
+
+## Label
+
+`Label` is a `label` element build based on the top of `Text` component with added ability to add
+pointer cursor or disabled style.
+
+### Usage
+
+```tsx
+<Label hasPointerCursor disabled={true}>
+  Text here...
+</Label>
+```
+
+<ExampleCodeBlock code={Label} />
+
+### Label Props
+
+In addition to `Box` component props, `Label` has the following props:
+
+<ArgsTable of={LabelComponent} />
+
+## Type Level Components
+
+Type Level Components, `TypeSubtextLevel`, `TypeBodyLevel`, `TypeHeadingLevel` and `TypeTitleLevel`,
+apply type token styles to text element. Instead of passing type level token you can use one of Type
+Level Components, like in example below:
+
+```tsx
+// v7
+<p css={{...type.levels.subtext.small}}>Small Subtext Text</p>
+
+// v8
+<TypeSubtextLevel size="small">Small Subtext Text</TypeSubtextLevel>
+```
+
+### Props
+
+In addition to `Box` component props, each type level components has the following props:
+
+<ArgsTable of={TypeLevelComponent} />
+
+### TypeBodyLevel
+
+`TypeBodyLevel` component renders text with `type.levels.body[size]` styles.
+
+```tsx
+<TypeBodyLevel size="small">Small Body Text</TypeBodyLevel>
+```
+
+<ExampleCodeBlock code={TypeLevelBody} />
+
+### TypeHeadingLevel
+
+`TypeHeadingLevel` component renders text with `type.levels.heading[size]` styles.
+
+```tsx
+<TypeHeadingLevel size="small">Small Heading Text</TypeHeadingLevel>
+```
+
+<ExampleCodeBlock code={TypeLevelHeading} />
+
+### TypeSubtextLevel
+
+`TypeSubtextLevel` component renders text with `type.levels.subtext[size]` styles.
+
+```tsx
+<TypeSubtextLevel size="small">Small Subtext Text</TypeSubtextLevel>
+```
+
+<ExampleCodeBlock code={TypeLevelSubtext} />
+
+### TypeTitleLevel
+
+`TypeTitleLevel` component renders text with `type.levels.title[size]` styles.
+
```

Comment:
Should we add any usage guidance of when to use this since this and heading are similar? Maybe add usage of when to use each component?

Reply:
We can to do it after getting clarification on this difference, it will require to extend of usage of type tokens. But now we can keep that we have.

Reply:
makes sense!